### DISCUSSION TITLE
Warning [12]: The following signature will be excluded if using Maximum Likelihood. Why?

### DISCUSSION THREAD

**Discussion by darrensterling** [semiautomaticgit/SemiAutomaticClassificationPlugin#253] [Q&A] [answered]
Getting the following error while doing a preview.   I've gone through the settings and confirmed that I do not have Maximum Likelihood on.  Is this just a warning, or is it referring to something I'm doing incorrectly?

The result i'm seeing is that I never see the preview region, instead I end up with a long list of the same warning.

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @darrensterling , yes it is just a warning that you can ignore if not using Maximum Likelihood
